### DISCUSSION TITLE
Is there anyway to hide the variables in the comment for TF Plan?

### DISCUSSION THREAD

**Discussion by jstuessy** [dflook/terraform-github-actions#252] [Q&A]
We have setup the terraform plan job to post the plan into the PR, however it shows all of the variables that were used in the job. I wanted to see if there is a way to have those hidden from the output comment.

**Comment by dflook** [comment]
If you set a label for the plan using the label input of the dflook/terraform-plan action, the variables will not be included in the comment.
You would also need to set the label on any matching dflook/terraform-apply action.
